doc/api/Gateway.md

```diff
@@ -150,7 +150,7 @@ Examples
 
 ###### Request
 ```
-/gateway/disconnect/123.456.789.0:123
+/gateway/disconnect/23.239.14.98:9981
```

Comment:
I don't think we should be using bootstrap addresses in the docs. The old IP was clearly a placeholder; this is not.